internal/internal_worker.go

```diff
@@ -1608,7 +1608,14 @@ func NewAggregatedWorker(client *WorkflowClient, taskQueue string, options Worke
 		tagNamespace, client.namespace,
 		tagTaskQueue, taskQueue,
 		tagWorkerID, workerParams.Identity,
+		tagBuildID, workerParams.getBuildID(),
```

Comment:
Are you adding `tagBuildID ` twice here?

Reply:
fixed.